src/main/java/seedu/address/logic/parser/AddCommandParser.java

```diff
@@ -51,9 +54,11 @@ public AddCommand parse(String args) throws ParseException {
         Email email = ParserUtil.parseEmail(argMultimap.getValue(PREFIX_EMAIL).get());
         Address address = ParserUtil.parseAddress(argMultimap.getValue(PREFIX_ADDRESS).get());
         BirthDate birthDate = ParserUtil.parseBirthDate(argMultimap.getValue(PREFIX_BIRTH_DATE).get());
+        RecentDate date = ParserUtil.parseContactedDate(argMultimap.getValue(PREFIX_DATE).get());
+        Description description = new Description("");
```

Comment:
Consider setting a default description when you initialise, since we the idea is that when you first add someone as a contact, there will be interaction